website/src/docs/uppy.md

```diff
@@ -339,9 +339,11 @@ uppy.addFile({
 
 `addFile` gives an error if the file cannot be added, either because `onBeforeFileAdded(file)` gave an error, or because `uppy.opts.restrictions` checks failed.
 
+If you try to add a file that already exists, `addFile` will throw an error. Unless that duplicate file was dropped/selected from a folder — duplicate files from different folders are allowed, when selected with that folder. This is because we add `file.meta.relativePath` to the `file.id`.
```

Comment:
When we select files from different folders there is no `.relativePath`, we only have `.relativePath` on drop.

Reply:
Right, I tried to mean the future, when we might add the “Select folder” button to the Dashboard.

Reply:
Tweaked.